languages/onoffice-de_DE.po

```diff
@@ -343,12 +343,18 @@ msgstr ""
 "Die Detailansicht wird gerade nicht verwendet. Fügen Sie diesen Code auf "
 "einer Seite ein, um die Detailansicht dort zu sehen:"
 
-#: plugin/Gui/AdminPageEstateDetail.php:161
-#: plugin/Gui/AdminPageEstateDetail.php:176
 #: plugin/Gui/AdminPageSettingsBase.php:177
 msgid "Fields"
 msgstr "Felder"
 
+#: plugin/Gui/AdminPageEstateDetail.php:161
+msgid "Contact person fields"
+msgstr "Felder Ansprechpartner"
+
+#: plugin/Gui/AdminPageEstateDetail.php:176
+msgid "Real estate data fields"
```

Comment:
The wording is different between "Contact person fields" and "Real estate data fields". I would prefer if the real estate variant would be "Real estate fields". Also, if this is a heading, I would prefer if the first character of every word was uppercase for the English language.

For the German ones: Wouldn't "Ansprechpartner-Felder" and "Immobilien-Felder" sound better?